README.md

```diff
@@ -1,59 +1,55 @@
 # pygame-web.github.io
-CDN root used by [pygbag](https://pypi.org/project/pygbag/)
 
+This is the CDN root used by [Pygbag](https://pypi.org/project/pygbag/) ([Source code](https://github.com/pygame-web/pygbag)/[Old runtimes](https://github.com/pygame-web/archives)) and the site of its [wiki](https://pygame-web.github.io/wiki/pygbag).
 
-[Github repo](https://github.com/pygame-web/pygbag), and [older runtimes](https://github.com/pygame-web/archives)
+Pygbag does not track usage at all, not even for statistical purposes. If you like it, please [star](https://github.com/pygame-web/pygbag/stargazers) the repo!
 
-This software does not track usage at all, not even for statistics purpose so if you like it :
+Check out some [demos](#demos-on-itchio) before you start! 
 
-please do not forget to [:star::star: Star :star::star:](https://github.com/pygame-web/pygbag/stargazers) it !
+**Work in progress, pull requests welcomed, feel free to propose links to games or tutorials, please contribute!!!**
 
-Stop talking i want to see some [Demos](https://pygame-web.github.io/#demos-on-itchio-) before reading everything.
+## Important points
 
-# Pygbag WIKI
+Read Pygbag's [project description](https://pypi.org/project/pygbag/) for a more detailed overview. A full packaging guide can be found [here](https://pygame-web.github.io/wiki/pygbag/).
 
-<ins>Do NOT forget to read the coding section for WASM. You WILL have to change a few lines to classic python/pygame code.</ins>
+### General
```

Comment:
This is not "general" this is "important poinst" sections title were swapped. all keep some emphasis (really not blinking red but close ;) ) on that so it gets read for sure

Reply:
ie if only one block was left to keep, it would be 
![shot-2024-01-18_1705550541](https://github.com/pygame-web/pygame-web.github.io/assets/16009100/c7d65272-6ea7-49a6-aa37-e0991624bb7d)

Reply:
What do you mean by “coding section for WASM”? Is it /wiki/python-wasm?

Reply:
yeah [MANDATORY main loop (async)] part, should make a link to it maybe